scrypto-unit/src/test_runner.rs

```diff
@@ -252,6 +252,7 @@ pub struct TestRunnerBuilder<E, D> {
     custom_database: D,
     trace: bool,
     state_hashing: bool,
+    collect_events: bool,
```

Comment:
Does this need to be a configuration option? Why not just enable it for all tests?

Reply:
Will keep it in for now